packages/example-project/package.json

```diff
@@ -10,13 +10,15 @@
 		"build": "chayns-toolkit build",
 		"dev": "chayns-toolkit dev",
 		"lint": "chayns-toolkit lint",
-		"test": "yarn lint"
+		"test": "yarn lint",
+		"jest": "chayns-toolkit test"
 	},
 	"eslintConfig": {
 		"extends": "@chayns-toolkit"
 	},
 	"dependencies": {
 		"chayns-components": "^4.20.25",
+		"@testing-library/react": "^11.2.3",
```

Comment:
`@testing-library/react` should probably be under `devDependencies`. Not that it would make a difference technically, but I think it makes sense for web projects to list everything that lands in the bundle as `dependencies` and everything else as `devDependencies`.